src/logger/stream-helpers.ts

```diff
@@ -34,7 +36,7 @@ export class SafeStreamHandler {
 
   protected writeStream(message: string): void {
     if (!this.ready) {
-      console.warn(`Stream busy. Some writes may be dropped.`);
+      console.warn(`Stream busy: ${this.name}. Write may be dropped: "${message}"`);
```

Comment:
don't we know definitively that the message *will* be dropped?  Should we re-word to drop the "may"?